OptimizelySDK/Data Model/Audience/UserAttribute.swift

```diff
@@ -97,7 +97,7 @@ extension UserAttribute {
 
         // invalid match - parsed for forward compatibility only (but evaluation fails)
         guard let matchFinal = matchSupported else {
-            throw OptimizelyError.userAttributeInvalidMatch(self.match ?? "empty")
+            throw OptimizelyError.userAttributeInvalidMatch(self.match!)
```

Comment:
there is no guarantee that match is not nil.  so either bind it or default it but not bang.

Reply:
nil match case is filtered out by "matchSupported", so safe for bang to remove unused code.
Will refactor the code to make safety more clear.